flink-runtime/src/main/java/org/apache/flink/runtime/asyncprocessing/operators/AbstractAsyncStateStreamOperator.java

```diff
@@ -212,19 +209,10 @@ protected <N, S extends State, T> S getOrCreateKeyedState(
     }
 
     @Override
-    public final OperatorSnapshotFutures snapshotState(
```

Comment:
Should the `AbstractAsyncStateStreamOperatorTest#testCheckpointDrain` be modified together?

Reply:
Thanks for the reminder! Modified accordingly.